packages/lit-dev-content/samples/examples/react-custom-event/react.ts

```diff
@@ -0,0 +1,12 @@
+declare global {
+  interface Window {
+    React: any;
+    ReactDOM: any;
+  }
+}
+
+// https://github.com/facebook/react/issues/10021
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+
+export { React, ReactDOM };
```

Comment:
same comments from `react.ts` in other playgrounds

Reply:
done!